### PR TITLE
[sc-2314] Add a delete account action UI (2)

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

**Review by ebrehault — APPROVED** [review]

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<pa-modal-dialog>
+  <pa-modal-title>
+    <h1>{{ 'account.delete_account_confirm_title' | translate }}</h1>
+  </pa-modal-title>
+  <pa-modal-content>
+    <div>
+      <label>{{ 'account.delete_account_confirm' | translate:{ account: (account | async)?.title } }}</label>
+      <pa-checkbox [(value)]="deleteAccount">{{ 'generic.yes' | translate }}</pa-checkbox>
```
Comment (mpellerin42) [inline apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html:8]: We agreed with Anna to replace the yes by just a checkbox with a label like "I want to delete the [name] account and all the data on it." (I see she didn't updated the figma file, but she seemed to like this idea in the comment)

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+    <div class="delete-user"
+         *ngIf="showDeleteUser">
+      <label for="group-delete-user">{{ 'account.delete_user_confirm' | translate }}</label>
+      <pa-radio-group [(value)]="deleteUser"
+                      id="group-delete-user">
+        <div>
```
Comment (mpellerin42) [inline apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html:15]: Do you really need this extra div?

File: apps/dashboard/src/app/setup/farewell/farewell.component.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<div class="farewell">
+  <div class="message-container">
+    <div class="message">
+
+      <h1>{{ 'farewell.title' | translate }}</h1>
+      <p>{{ 'farewell.message_1' | translate: { username: (userPrefs | async)?.name || '' } }}</p>
```
[the remark this answers is not in the document]
Comment (operramon) [reply apps/dashboard/src/app/setup/farewell/farewell.component.html]: Yes, a single message seems more simple, fixed.

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html (hunk 1 above)
Comment (operramon) [reply apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html]: Fixed

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html (hunk 2 above)
Comment (operramon) [reply apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.html]: 👍 It wasn't really necessary

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.scss [hunk 4]
```diff
@@ -10,12 +10,12 @@ label {
 }
 
 .delete-user {
-  margin: rhythm(4) 0;
+  margin: rhythm(4) 0 rhythm(3) 0;
 }
 
-pa-radio-group > div {
```
Comment (mpellerin42) [inline apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.scss]: oh I missed this on my first review! you don't need to change your new style for this, but just for you to know, you could have done what you wanted without the `div`, by adding an `!important` on the display property, like:

```scss
pa-radio-group {
  display: flex !important;
  gap: rhythm(5);
}
```

**Review by mpellerin42 — APPROVED** [review]

File: apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.scss (hunk 4 above)
Comment (mpellerin42) [reply apps/dashboard/src/app/account/account-manage/account-delete/account-delete.component.scss]: I should try to find another way to set the default display block on radio group so we don't need this `!important` though

**Pull request merged**